packages/create-drupal-decoupled/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@octahedroid/create-drupal-decoupled",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
this is not needed, as the changesets tool will do it for you after merging this PR 👀